app/scripts/controllers/phishing-controller.js

```diff
@@ -1,30 +1,8 @@
 import { PhishingController } from '@metamask/controllers'
 
-const PhishingDetector = require('eth-phishing-detect/src/detector')
```

Comment:
i think this dependency can be removed? this seems to be the only file that uses it